save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -96,16 +98,26 @@ class DownloadFilesController(
     /**
      * @param file image to be uploaded
      * @param owner owner name
+     * @param isOrganization flag of organization
      * @return [Mono] with response
      */
     @Suppress("UnsafeCallOnNullableType")
     @PostMapping(value = ["/api/image/upload"], consumes = [MediaType.MULTIPART_FORM_DATA_VALUE])
-    fun uploadImage(@RequestPart("file") file: Mono<FilePart>, @RequestParam owner: String) =
-            additionalToolsFileSystemRepository.saveImage(file, owner).map { imageInfo ->
+    fun uploadImage(
+        @RequestPart("file") file: Mono<FilePart>,
+        @RequestParam owner: String,
+        @RequestParam(required = false) isOrganization: Boolean = true
+    ) =
+            additionalToolsFileSystemRepository.saveImage(file, owner, isOrganization).map { imageInfo ->
                 ResponseEntity.status(
                     imageInfo.path?.let {
-                        organizationService.saveAvatar(owner, it)
-                        HttpStatus.OK
+                        if (isOrganization) {
```

Comment:
I think this is too much code for calling inside `ResponseEntity` creation. Please extract saving before creating `ResponseEntity`

Reply:
Fixed